corehq/apps/accounting/user_text.py

```diff
@@ -54,6 +54,7 @@
     FeatureType.SMS: _("SMS Messages"),
     FeatureType.USER: _("Mobile Workers"),
     FeatureType.WEB_USER: _("Web Users"),
+    FeatureType.FORM_SUBMITTING_MOBILE_WORKER: _("Form-Submitting Mobile Workers"),
```

Comment:
Just want to highlight that it's weird both `FEATURE_TYPE_TO_NAME` and `get_feature_name` exist. I'm not sure one translating 'SMS' to 'SMS Messages' while the other translates it to 'SMS' is a feature or a bug. Probably out of scope

Reply:
Thanks, I thought it was weird too but didn't look into it. It seems like `get_feature_name` is just used by the "Feature Summary" on the Current Subscription page, where `FEATURE_TYPE_TO_NAME` is for the user-facing description of a Software Plan Version (which is used in a bunch of places). I'm inclined to just point `get_feature_name` to the `FEATURE_TYPE_TO_NAME` dict, seeing that the function is only used in one place.

Reply:
https://github.com/dimagi/commcare-hq/pull/34868/commits/a5ae823f1907ca4a6da4ce4874c9ea20df452e4b